blockly/src/antlr/blocklyConditionVisitor.java

```diff
@@ -14,6 +14,8 @@
 @SuppressWarnings("CheckReturnValue")
 public class blocklyConditionVisitor extends blocklyBaseVisitor<INode> {
```

Comment:
WOW. Klassen in Java klein geschrieben?! Krass.

@Flamtky Kannst Du bitte ein Ticket aufmachen? Das muss refactored werden. (Nur nicht hier.)

Reply:
😅Hatte ich auch mir gedacht. Aber soll ich das wirklich nicht kurz hier ändern? Die Klasse wird nur in Server verwendet.

Reply:
btw. weiß du den Grund für `@SuppressWarnings("CheckReturnValue")`

Reply:
> 😅Hatte ich auch mir gedacht. 
> 
> 
> 
> Aber soll ich das wirklich nicht kurz hier ändern? Die Klasse wird nur in Server verwendet.

hmmm. ich bin immer kein großer freund von so mörder-pr's. 

aber wenn es wirklich nur diese eine klasse ist, dann von mir aus :)

Reply:
> btw. weiß du den Grund für `@SuppressWarnings("CheckReturnValue")`
> 
> 

keine idee. nimms mal raus - was sagt dann das tooling (build, check)?

Reply:
> keine idee. nimms mal raus - was sagt dann das tooling (build, check)?

Hat nix geändert. Keine Warnings und auch keine bei check und build